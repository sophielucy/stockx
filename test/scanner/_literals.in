34
34.5
myfunc
thisisanid
first_student
"stringlit"
"hello world!"
