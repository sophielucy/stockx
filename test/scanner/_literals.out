INT_LITERAL
FLOAT_LITERAL
ID
ID
ID
STRING_LITERAL
STRING_LITERAL
